app/models/rejection.rb

```diff
@@ -11,4 +11,12 @@ class Rejection < ActiveRecord::Base
   belongs_to :visit
 
   validates :reason, inclusion: { in: REASONS }
+
+  def pvo_possible?
```

Comment:
The "P" in PVO is "privileged", but it's still an opaque identifier without context.